osmdroid-android/src/main/java/org/osmdroid/LocationListenerProxy.java

```diff
@@ -6,58 +6,56 @@
-		mLocationManager = pLocationManager;
-	}
-
-	public boolean startListening(final LocationListener pListener, final long pUpdateTime,
-			final float pUpdateDistance) {
-		boolean result = false;
-		mListener = pListener;
-		for (final String provider : mLocationManager.getProviders(true)) {
-			if (LocationManager.GPS_PROVIDER.equals(provider)
-					|| LocationManager.NETWORK_PROVIDER.equals(provider)) {
-				result = true;
-				mLocationManager.requestLocationUpdates(provider, pUpdateTime, pUpdateDistance,
-						this);
-			}
-		}
-		return result;
-	}
-
-	public void stopListening() {
-		mListener = null;
-		mLocationManager.removeUpdates(this);
-	}
-
-	@Override
-	public void onLocationChanged(final Location arg0) {
-		if (mListener != null) {
-			mListener.onLocationChanged(arg0);
-		}
-	}
-
-	@Override
-	public void onProviderDisabled(final String arg0) {
-		if (mListener != null) {
-			mListener.onProviderDisabled(arg0);
-		}
-	}
-
-	@Override
-	public void onProviderEnabled(final String arg0) {
-		if (mListener != null) {
-			mListener.onProviderEnabled(arg0);
-		}
-	}
-
-	@Override
-	public void onStatusChanged(final String arg0, final int arg1, final Bundle arg2) {
-		if (mListener != null) {
-			mListener.onStatusChanged(arg0, arg1, arg2);
-		}
-	}
+    private final LocationManager mLocationManager;
+    private LocationListener mListener = null;
+
+    public LocationListenerProxy(final LocationManager pLocationManager) {
+        mLocationManager = pLocationManager;
+    }
+
+    public boolean startListening(final LocationListener pListener, final long pUpdateTime,
+                                  final float pUpdateDistance) {
+        boolean result = false;
+        mListener = pListener;
+        for (final String provider : mLocationManager.getProviders(true)) {
+            if (LocationManager.GPS_PROVIDER.equals(provider) || LocationManager.NETWORK_PROVIDER.equals(provider)) {
+                result = true;
+                mLocationManager.requestLocationUpdates(provider, pUpdateTime, pUpdateDistance, this);
+            }
+        }
+        return result;
+    }
+
+    public void stopListening() {
+        mListener = null;
+        mLocationManager.removeUpdates(this);
+    }
+
+    @Override
+    public void onLocationChanged(final Location arg0) {
+        if (mListener != null) {
+            mListener.onLocationChanged(arg0);
+        }
+    }
+
+    @Override
+    public void onProviderDisabled(final String arg0) {
+        if (mListener != null) {
+            mListener.onProviderDisabled(arg0);
+        }
+    }
+
+    @Override
+    public void onProviderEnabled(final String arg0) {
+        if (mListener != null) {
+            mListener.onProviderEnabled(arg0);
+        }
+    }
+
+    @Override
+    public void onStatusChanged(final String arg0, final int arg1, final Bundle arg2) {
+        if (mListener != null) {
+            mListener.onStatusChanged(arg0, arg1, arg2);
+        }
+    }
```

Comment:
is this change just a reformat or was there any substantive changes?

Reply:
format

Reply:
This repo would need a reformat of all code, but when I do it, I will touch almost every file 
.... on the level you trust, it's impossible for me to do this step 😄 

This is a pre-task to format all https://github.com/osmdroid/osmdroid/pull/1620